### DISCUSSION TITLE
Solution to remove console log information

### DISCUSSION THREAD

**Discussion by achKiran26** [nolimits4web/swiper#5921] [Q&A]
Hi Everyone, 
We are seeing console log data popping up when using swiper js. We have installed swiper via node packages.
Is there a way that we can remove these console log data.

<img width="1728" alt="Screenshot 2022-07-29 at 9 54 00 PM" src="https://user-images.githubusercontent.com/96536688/181803113-d3657780-8fed-4514-acb4-0d57bb755208.png">

Kindly looks into this.
Thanks

**Comment by nolimits4web** [comment]
These console.logs are coming not from Swiper
